client/src/ts/components/Beatmap/Statistics.tsx

```diff
@@ -85,3 +101,18 @@ export const BeatmapStats: FunctionComponent<IProps> = ({
 const isFullMap: (map: IMapStats | IBeatmap) => map is IBeatmap = map => {
   return (map as IBeatmap).downloadURL !== undefined
 }
+
+const convertSecondsToTime: (duration: number) => string = duration => {
+  const hours = Math.trunc(duration / 3600)
+  const minutes = Math.trunc((duration % 3600) / 60)
+  const seconds = Math.trunc(duration % 60)
+
+  let formattedDuration = ''
```

Comment:
Avoid using `let`.
Try to rewrite with a more functional approach

Reply:
I took a stab at it.  If you had something else in mind, please provide an example

Reply:
Refactored code looks good 👍